resources/shuttle-shared-db.mdx

```diff
@@ -12,4 +12,25 @@ Add `shuttle-shared-db` to the dependencies for your service. Every type of shar
 | Postgres | postgres     | shuttle_shared_db::Postgres |
 | MongoDB  | mongodb      | shuttle_shared_db::MongoDb  |
 
+
+## Example
+The post function using the rocket framework is as follows:
```

Comment:
This example doesn't really show how to use the shuttle-shared-db to provision and access a db, rather it shows how to use a sql pool in rocket handlers. Maybe we should rather use a shuttle axum main function here, my suggestion below. The code explaining the example should be changed, it can be similar to the aws rds example explanation.

````rust
#[shuttle_service::main]
async fn axum(#[shuttle_shared_db::Postgres] pool: sqlx::PgPool) -> shuttle_service::ShuttleAxum {
    pool.execute(include_str!("../schema.sql"))
        .await
        .map_err(shuttle_service::error::CustomError::new)?;

    let app = Router::new().route("/hello", get(hello_world)).layer(Extension(pool));

    let sync_wrapper = SyncWrapper::new(app);
    Ok(sync_wrapper)
}